polaris-react/src/components/IndexFilters/components/UpdateButtons/UpdateButtons.tsx

```diff
@@ -87,7 +87,7 @@ export function UpdateButtons({
 
   const saveButton = (
     <Button
-      size="micro"
+      size="slim"
```

Comment:
Should this change be behind the beta flag (like we have for primary/plain props) so that it still renders a micro button without the beta flag? Same with the cancel button